Phonebook.Frontend/src/app/pages/settings/settings.component.html

```diff
@@ -41,17 +42,15 @@ <h2 i18n="SettingsComponent|Color Theme SubTitle@@SettingsComponentSubTitleColor
           </mat-form-field>
         </mat-list-item>
       </mat-list>
+      <mat-card-subtitle class="subtitleSetting" style="margin-top: 30px" i18n="SettingsComponent|Subtitle Dashboard of the settings page@@SettingsComponentSubtitleDashboard">
```

Comment:
why do you don't write this style to the class?

Reply:
and you can also use a predefined class. I am not sure, but we have something like "pb-margin-top-20"